.github/workflows/build-container.yml

```diff
@@ -620,9 +620,44 @@ jobs:
           make down
           make clean
 
+  atom:
+    needs: [build, build-ceph]
+    runs-on: ibmcloud-1
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Download container images
+        uses: actions/download-artifact@v4
+        with:
+          pattern: container_images*
+          merge-multiple: true
+
+      - name: Nvmeof Atom cleanup env
+        run: /home/cephnvme/atomEnvCleanup.sh
```

Comment:
Could we add /home/cephnvme/*sh scripts to this repo as part of this PR?

Reply:
those scripts are saved in the atom project.
moving the tests and the scripts to our project is about to be discussed soon with Aviv.